src/main/java/seedu/address/ui/RegistrationWindow.java

```diff
@@ -30,6 +28,7 @@
     private TextField addressTextField;
     @FXML
     private Label label;
+    private Integer registerFlag;
```

Comment:
Same as above, probably better and clearer to use an enum.

Reference: https://nuscs2113-ay1819s1.github.io/website/se-book-adapted/chapters/javaTools.html#enums